src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -73,14 +68,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
     String comment = request.getParameter(CommentDataManager.COMMENT_PROPERTY_KEY);
-    if(comment == null) {
-      System.out.println("Comment was not entered");
+    if (comment == null) {
+      System.out.println("Comment was not entered.");
```

Comment:
I think it would be better to send an error message in the response instead of printing it  with System.out.println(). eg:
```   response.sendError(HttpServletResponse.SC_BAD_REQUEST,
          "Comment was not entered.");```